build-packages/magento-scripts/lib/util/prefix.js

```diff
@@ -8,6 +8,8 @@ const getPrefix = () => {
 
     if (!projectInGlobalConfig || !projectInGlobalConfig.createdAt) {
         const createdAt = Math.floor(Date.now() / 1000).toString();
+        process.isLegacy = 1;
```

Comment:
Project is legacy means that the user used an older version of `magento-scripts` that didn't have prefixes and we try not to break links with docker volumes and containers because of that.

Later, if no `prefix: false` option is set in the config file, the prefix will be added in `setPrefix` function.

SO, I suggest renaming this boolean to something more related to our case, like `isFirstStart` :thinking: